libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -280,14 +280,27 @@ public void onCreate(Bundle savedInstanceState) {
             mHideLoginWithSiteOption = args.getBoolean(ARG_HIDE_LOGIN_BY_SITE_OPTION, false);
             mLoginSiteUrl = args.getString(ARG_LOGIN_SITE_URL, "");
         }
+    }
 
+    @Override
+    public void onStart() {
+        super.onStart();
         mGoogleApiClient = new GoogleApiClient.Builder(getActivity())
                 .addConnectionCallbacks(LoginEmailFragment.this)
                 .enableAutoManage(getActivity(), GOOGLE_API_CLIENT_ID, LoginEmailFragment.this)
                 .addApi(Auth.CREDENTIALS_API)
                 .build();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        mGoogleApiClient.stopAutoManage(getActivity());
+        if (mGoogleApiClient != null && mGoogleApiClient.isConnected()) {
```

Comment:
Same comment as [here](https://github.com/wordpress-mobile/WordPress-Android/pull/10523/files#r329371454). Maybe I would also consider to add same null check logic here for consistency. Let me know what you think 😊.